xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/src/main/resources/Main/Tags.xml

```diff
@@ -85,7 +85,7 @@ $xwiki.ssx.use('Main.Tags')##
       === $services.localization.render('xe.tag.alldocs', ["//${wikiEscapedTag}//"]) ===
 
       #if ($list.size()&gt; 0)
-        {{html}}#displayDocumentList($list false $blacklistedSpaces){{/html}}
+        {{html}}#displayDocumentList($list false $NULL){{/html}}
```

Comment:
Couldn't you omit the parameter?

```suggestion
        {{html}}#displayDocumentList($list false){{/html}}
```

Reply:
I generally prefer to specify all parameters: I'm always a bit afraid that my call was actually calling another macro with same name and fewer arguments...

Reply:
I don't think that's possible, if you declare two velocity macros with the same name, the second one will take precedence, regardless of its signature.

Reply:
ok, well I still prefer to be explicit on the number of parameters I think.